Source/Core/BoundingRectangle.js

```diff
@@ -79,12 +79,8 @@ define([
      */
     BoundingRectangle.pack = function(value, array, startingIndex) {
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(value)) {
-            throw new DeveloperError('value is required');
-        }
-        if (!defined(array)) {
-            throw new DeveloperError('array is required');
-        }
+        Check.typeOf.object(value, "value");
+        Check.defined(array, "array");
```

Comment:
Throughout use single quotes instead of double quotes. Otherwise all looks good.